README.md

```diff
@@ -44,15 +44,29 @@
 
 To name functions, you can use some keywords that everyone will understand, like those ones :
 
-`updateSomething` : should perform server-side updates
+`fetchSomething` : should access data by making a server side call
 
 `getSomething` : should access data without making a server side call
 
-`fetchSomething` : should access data by making a server side call
+`findSomething` : like the get, but with arguments
 
 `normalizeSomething` : should transform data to conform to a given schema or to make sure that is coherent with other documents
 
-`harmonizeSomething` : should tweak data a bit to work well with other items
+`saveSomething` : creates or updates a document
+
+`updateSomething` : should perform server-side updates
```

Comment:
Since cozy-client doesn't expose any `update` method, I'm not sur about this one.